source/module_hamilt_lcao/module_tddft/bandenergy.cpp

```diff
@@ -0,0 +1,151 @@
+#include "bandenergy.h"
```

Comment:
In these files, git history can not find the real history of these code. you can choose some methods:
1. add annotations for these files, list which file and which commit can find the history code.
2. add UTs for functions in these files.

Reply:
Annotations are added already. UTs will be added in next PR.